packages/react/src/VideoPlayer/VideoPlayer.features.stories.tsx

```diff
@@ -11,14 +12,85 @@ export default {
 
 export const WithPoster = () => (
   <VideoPlayer poster={posterImage} title="GitHub media player">
-    <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
   </VideoPlayer>
 )
 
 export const WithoutBranding = () => (
-  <VideoPlayer branding={false} title="GitHub media player">
-    <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+  <VideoPlayer showBranding={false} title="GitHub media player">
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const WithoutTitle = () => (
+  <VideoPlayer showTitle={false} title="GitHub media player">
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const WithCustomControls = () => (
+  <VideoPlayer
+    title="GitHub media player"
+    renderControls={() => (
+      <ControlsBar>
+        <PlayPauseButton />
+        <SeekControl />
+      </ControlsBar>
+    )}
+  >
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
```

Comment:
It seems like play/pause and seek controls are enabled by default, so this story feels a little redundant. Could we consider flipping this to 'No controls' instead?

Reply:
Good point, it's poorly named and not very clear. I was trying to show that the user can mix and match the controls by omitting some, including some, and even writing their own. I think that's a valuable thing to demo, but I'll look at making that clearer and also add in a no controls example too

Reply:
Resolved in 4873e4713bad9c9ce1780008cf36c994d2f7053d